app/classifier/tasks/components/TaskInputField/components/TaskInputLabel/TaskInputLabel.jsx

```diff
@@ -0,0 +1,71 @@
+import { Markdown } from 'markdownz';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+function howShouldTheLabelBeAligned(label, labelIcon) {
+  if ((label && label.includes('<img')) || (label && labelIcon)) {
```

Comment:
Is the label HTML or Markdown? The label strings in task translations are Markdown, and images are in square brackets (I think.)

Reply:
Good point. I'll check on that.

Reply:
Maybe the Markdown was parsed further up the chain, in which case ignore my comment.